jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -252,11 +254,21 @@ public JibContainerBuilder createJibContainerBuilder(
       JavaContainerBuilder javaContainerBuilder, ContainerizingMode containerizingMode)
       throws IOException {
     try {
+
+      Stream<Artifact> mavenProjectArtifactStream =
+          session.getProjects().stream().map(MavenProject::getArtifact);
```

Comment:
nit: for consistency with Gradle, let's say
```suggestion
      Set<Artifact> projectDependencies =
          session.getProjects().stream().map(MavenProject::getArtifact).collect(Collectors.toSet());
```